documentation/homework/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import Item from './components/Todo-item'
+import './App.css'
+import TodoList from './TodosList.json'
+
+
+const Content = () => {
```

Comment:
Normally it's seen as bad practice to have more than one React component per JavaScript file. You could however put this as a class property of `App` itself, eg:

    renderContent = () => ...your JSX logic

    render() { <div>{this.renderContent()}</div> }